internal/storage/bucket/migrations/27-fix-invalid-pcv/up.sql

```diff
@@ -0,0 +1,59 @@
+do $$
```

Comment:
Copy/paste of the migration v19 (it will re run it) but with the fix (for clients being already migrated)